src/javascript/app/Modules/Contract/Components/LastDigitPrediction/last-digit-pointer.jsx

```diff
@@ -1,7 +1,7 @@
-import classNames        from 'classnames';
-import PropTypes         from 'prop-types';
-import React             from 'react';
-import { IconPriceMove } from 'Assets/Trading/icon-price-move.jsx';
+import classNames from 'classnames';
+import PropTypes  from 'prop-types';
+import React      from 'react';
+import Icon       from 'Assets/Trading/icon-price-move.jsx';
```

Comment:
i thought we can make the icon registered in singular index file

```suggestion
import Icon       from 'Assets/Common';
```